views/index.ejs

```diff
@@ -61,7 +61,8 @@
       <%- include navbar %>
       <%- include overlay %>
       <%- include title %>
-      <div class="g mnone">
+      //SplitJS
+      <div class="g mnone" ng-controller="diSplitJs">
         <%- include editor-headers %>
         <%- include editor %>
         <%- include preview %>
```

Comment:
I would write this section like this
```html
<div class="g mnone">
  <%- include editor-headers %>
  <div id="split-area">
    <%- include editor %>
    <%- include preview %>
  </div>
</div>
```

Which should help with the common parent problem.